scanner/src/components/Common/Title/index.svelte

```diff
@@ -0,0 +1,16 @@
+<div class={`title ${$$props.class}`}>
+  <slot />
+</div>
+
+<style lang="scss">
+  .title {
+    font-size: 1rem;
+    font-weight: 500;
+    line-height: 1.5rem;
+    margin-bottom: 1rem;
+
+    &.no-bottom-margin {
```

Comment:
it's a bad pattern than you spread the styles across multiple places, `no-bottom-margin` is a class that you have to set outside, so it becomes a bit of fragile if you want to for example refactor class names later. and for the user of `Title` component the need to set this magic class is kind of undocumented - you have to look into `scss` of a Title to learn about this behaviour. Please add a boolean prop to this component instead and assign additional class name within the component, if the prop is set. Same applies for Subtitle component